README.md

```diff
@@ -139,31 +139,43 @@ signing transactions, working with accounts, assets, storing blocks in database
 There are 3 types of network. 
 Private net -- it’s the private one which you can run locally. Testnet and Mainnet where much of the nodes across the world now running. 
 NEO has a nice monitor where you can find particular node running in the blockchain network.
-<p align="center">
-<img src="https://gfycat.com/legitimatepertinentindianrhinoceros" width="300px" alt="network_monitor">
-</p>
+[Neo Monitor](http://monitor.cityofzion.io/)
 
 ## Workshop
 Now it’s time to run your private network. Connect neo-go node to it, write smart contract and deploy it. Let’s go!
 
 #### Step 1
-Run local privnet:
+Download neo-go and build it
```

Comment:
It'd be nice to keep instructions for neo-local setup, it has neo-scan built-in which is useful for demoing purposes. It also shows various node implementations interoperability.

Reply:
true, will rework this. Like user has a choice (two chairs) and could run neo-go or neo-local with neo-scan.